helm/cert-operator-chart/values.yaml

```diff
@@ -1 +1,2 @@
+labelSelector: ''
```

Comment:
Can we have this? I would like to get the values aligned with the actual config maps and flags. 

```
service: 
  crd: 
    labelSelector: ''
```